spec/features/forms/form_item/questionings_form_spec.rb

```diff
@@ -55,6 +56,20 @@
           expect_visible("hidden", true)
         end
 
+        it "should hide disabled option when metadata field has a value" do
```

Comment:
Shouldn't it be hiding _hidden_ instead of disabled when metadata has a value? Hidden is implied in that case, but disabled is not. You could still disable a metadata question if you don't want to collect data for it anymore.

Reply:
Ah good catch. Both were being hidden for metadata, I'll make sure `disabled` checkbox is always rendered.